test/TestConfig/settings.mk

```diff
@@ -211,6 +211,10 @@ else
 export JCL_VERSION:=$(JCL_VERSION)
 endif
 
+ifndef JAVA_IMPL
+export JAVA_IMPL:=openj9
+endif
+
```

Comment:
Please move this code to the beginning of the file, after to JAVA_VERSION. Say line 60.
It is better keep all exports together in the beginning of the mk.
Please add `@$(ECHO) set JAVA_IMPL to $(JAVA_IMPL)` to setup_% target

Reply:
Updated.